_life/money.md

```diff
@@ -5,29 +5,42 @@ category: life
+The short version:
+- PhD students get paid around 55k CHF/year, with small yearly increases, and paystubs are available on the internal [SESAME](https://sesame2.epfl.ch) tool.
+- To store this money, get an account at any Swiss bank, they are roughly equivalent in terms of services and typically have student discounts.
+- Get your bank's [TWINT](https://www.twint.ch/en/) app to easily send and receive money using phone numbers, you typically won't need a credit card.
+- You automatically save money for retirement as an employee in Switzerland, and you can save extra money in a "3a pillar" account which is tax-deductible up to a limit.
+- If you are on a B permit _and_ you make less than 120k _and_ your wealth is under 50k _and_ you have no undeclared revenue, taxes are taken automatically from your salary and you need not worry about them.
+- If you do not meet the criteria above, or if you have special deductions such as a 3a pillar or withheld taxes from investments or savings, you need to explicitly file your taxes, see below.
 
 
 ## Pay
 
-The pay for PhD students is set at the EPF level subject to the approval of the
-*Conseil des EPF* / *ETH Rat*. As of 2024, it is 54,550 CHF/year for the first
-year, and increases by 1,000 CHF/year in each of the 2nd, 3rd, and 4th years
+The pay for PhD students is set at the federal level subject to the approval of the ETH council.
+As of 2024, it is 54,550 CHF/year for the first year, and increases by 1,000 CHF/year in each of the 2nd, 3rd, and 4th years
 ([source](https://www.epfl.ch/campus/services/human-resources/en/basic-starting-salary-of-doctoral-assistants-and-postdocs/)).
 
+The main unusual factor is the "13th salary" system: your salary is divided into 13, not 12, parts,
+and the extra 1/13th being given to you partly in November and partly in December.
+The origins of this system are historically not so clear, but nowadays it’s basically a way for you to pay for Christmas presents and such.
+If you did not work the whole year, for instance because you started in September, or because you did an internship in the summer,
+the 13th salary is adjusted proportionally.
+
+Overall, expect to get around 3,200 CHF/month after all taxes starting in the first year, plus the 13th salary.
+Here’s an example breakdown for a first year monthly PhD student salary, for a non-November/December month (see above), **as of January 2023** (the amounts may have slightly changed since then):
+
+| Gross salary             | 4194.65 CHF |
+| First pillar (AVS)       | -222.30 CHF  |
+| Various other deductions | -83.75 CHF   |
+| Second pillar (Publica)  | -152.70 CHF  |
+| Income tax               | -251.25 CHF  |
+| **Net income**           | **3484.65 CHF** |
+
 Note that the 1,000 CHF/year increment:
-* applies every *calendar* year (not based on your ‘PhD years’), and three times
-  in total;
+* applies every *calendar* year, not "PhD year", three times in total;
```

Comment:
The "and" makes a semantic difference. Without it, this _could_ be parsed as 'three times in total every calendar year'.